framework/include/utils/Conversion.h

```diff
@@ -67,6 +67,11 @@ stringify(const T & t)
 
 // overloads for integer types where std::to_string gives the same result and is faster
 inline std::string
+stringify(bool v)
+{
+  return v ? "true" : "false";
+}
```

Comment:
this should get its own comment and not displace the comment applied to the integer overloads

Reply:
Good catch, just updated. Thanks!